cmd/earthly/base/buildkit.go

```diff
@@ -55,6 +55,8 @@ func (cli *CLI) ConfigureSatellite(cliCtx *cli.Context, cloudClient *cloud.Clien
 		cli.Flags().BuildkitdSettings.TLSCA = ""
 		cli.Flags().BuildkitdSettings.ServerTLSCert = ""
 		cli.Flags().BuildkitdSettings.ServerTLSKey = ""
+	} else {
+		cli.Console().Warnf("TLS has been disabled; this should never be done when connecting to Earthly's production API\n")
```

Comment:
nit: I think we can tell whether the user is trying to connect to the production API so the warning does not have to be displayed in localdev for example.

Reply:
I wanted to get a quick warning in place since there's been a user who struggled with this issue.

If this warning becomes an issue, we could address it at a later time, but I don't want this to become a distraction from our higher priority issues.

If you do know how to easily do this, feel free to open a PR (but I don't think it warrants spending much time on it)